ReactWindows/ReactNative.Net46/Touch/TouchHandler.cs

```diff
@@ -383,6 +429,58 @@ public override void Dispatch(RCTEventEmitter eventEmitter)
             }
         }
 
+        class PointerEnterExitEvent : Event
+        {
+            private readonly TouchEventType _touchEventType;
+
+            public PointerEnterExitEvent(TouchEventType touchEventType, int viewTag)
+                : base(viewTag, TimeSpan.FromTicks(Environment.TickCount))
+            {
+                _touchEventType = touchEventType;
+            }
+
+            public override string EventName
+            {
+                get
+                {
+                    return _touchEventType.GetJavaScriptEventName();
+                }
+            }
+
+            public override bool CanCoalesce
+            {
+                get
+                {
+                    return false;
+                }
+            }
+
+            public override void Dispatch(RCTEventEmitter eventEmitter)
+            {
+                var eventData = new JObject
+                {
+                    { "target", ViewTag },
+                };
+
+                var enterLeaveEventName = default(string);
```

Comment:
maybe make this a switch with a default case? using value absence here read weird to me.